src/main/resources/META-INF/mods.toml

```diff
@@ -21,4 +21,4 @@ The open-source Forge map mod, made by modders for modders.
     mandatory=true
     versionRange="[1.19.4,1.20)"
     ordering="NONE"
-    side="BOTH"
+    displayTest="NONE"
```

Comment:
"NONE" here means "go check what I declared in `BlazeMap.java` to see if the versioning matches". Don't blame me, blame Forge.

Reply:
wtf ??